lib/dbi/ogs-dbi.h

```diff
@@ -24,7 +24,9 @@
 
 #define OGS_DBI_INSIDE
 
+/* TODO: remove ogs-mongoc.h as soon the test cases don't depend on it */
```

Comment:
This commit in itself does nothing, so I'm sure you want to merge this into some other commit